projects/testing-library/src/lib/testing-library.ts

```diff
@@ -236,3 +239,12 @@ function addAutoImports({ imports, routes }: Pick<RenderComponentOptions<any>, '
 
   return [...imports, ...animations(), ...routing()];
 }
+
+if (typeof afterEach === 'function' && !process.env.ATL_SKIP_AUTO_CLEANUP) {
```

Comment:
When `process.env.ATL_SKIP_AUTO_CLEANUP` is going to be neccessary?

Reply:
It's to be consistent with the rest of the testing libraries